src/app/screens/TodoList/components/TodoInput/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Field, reduxForm, reset } from 'redux-form';
+import { TextInput } from 'react-native';
+
+import { actionCreators } from '../../../../../redux/toDos/actions';
+
+import styles from './styles';
+
+const FORM_NAME = 'todoInput';
+
+const renderInput = ({ input: { onChange, ...restInput }, onSubmitEditing }) => (
+  <TextInput onChangeText={onChange} {...restInput} onSubmitEditing={onSubmitEditing} />
```

Comment:
you cant use customTextInut component Here?

Reply:
I tried but I couldn't get it to work. This is as far as I made it, but the `submit` doesn't get any params
` <Field name={TODO_TEXT_FIELD} component={CustomTextInput} onSubmitEditing=handleSubmit(submit)} bottomBorder/> `